annotation-processor/src/main/java/org/hibernate/validator/ap/internal/util/Configuration.java

```diff
@@ -97,11 +98,12 @@ private Kind getDiagnosticKindOption(Map<String, String> options, Messager messa
 			}
 			catch (IllegalArgumentException e) {
 				messager.printMessage(
-						Kind.WARNING, MessageFormat.format(
-						"The given value {0} is no valid diagnostic kind. {1} will be used.",
-						diagnosticKindFromOptions,
-						DEFAULT_DIAGNOSTIC_KIND
-				)
+						Kind.WARNING, String.format(
+								Locale.ROOT,
```

Comment:
just as an idea, would be nice to hide these `Locale.ROOT` in some `StringHelper` util method. But as they are scattered all over the modules I cannot see a nice place for such method.

Reply:
Yeah, for now, I don't think it's worth it. We don't have thousands of calls and we don't add them regularly, mostly thanks to JBoss Logging.